maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/NodeHelper.java

```diff
@@ -29,27 +29,27 @@
 	private final MavenProject mavenProject;
 	
 	private final BuildPluginManager pluginManager;
+
+	private Path tempDir;
 	
 	public NodeHelper(MavenProject mavenProject,
 			MavenSession mavenSession,
 			BuildPluginManager pluginManager) {
 		this.mavenProject = mavenProject;
 		this.mavenSession = mavenSession;
 		this.pluginManager = pluginManager;
+		try {
+			this.tempDir = Files.createTempDirectory("openmrs-sdk-node");
```

Comment:
Node, NPM, and the cache all get cleaned-up but not the tempDir itself.

```suggestion
			this.tempDir = Files.createTempDirectory("openmrs-sdk-node");
			this.tempDir.toFile().deleteOnExit();
```